{"uid":"f249454378ef7a1e","name":"Check Dialog elements","fullName":"Check Dialog elements","historyId":"c8f6cb227b5890707d4cd3bcceed9e71","time":{"start":1680611777815,"stop":1680611778381,"duration":566},"status":"broken","statusMessage":"The following error originated from your test code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > window.confirm wasn't called within 1s\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.","statusTrace":"Error: The following error originated from your test code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > window.confirm wasn't called within 1s\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.\n    at eval (webpack:///../common/base.ts:708:23)","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"broken","statusMessage":"The following error originated from your test code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > window.confirm wasn't called within 1s\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.","statusTrace":"Error: The following error originated from your test code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > window.confirm wasn't called within 1s\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.\n    at eval (webpack:///../common/base.ts:708:23)","steps":[{"name":"\"before each\" hook","time":{"start":1680611777815,"stop":1680611778380,"duration":565},"status":"failed","statusMessage":"The following error originated from your test code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > window.confirm wasn't called within 1s\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.","statusTrace":"Error: The following error originated from your test code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > window.confirm wasn't called within 1s\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.\n    at eval (webpack:///../common/base.ts:708:23)","steps":[{"name":"visit http://localhost:3001","time":{"start":1680611777907,"stop":1680611778380,"duration":473},"status":"failed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"uncaught exception Error: window.confirm wasn't called within 1s","time":{"start":1680611778380,"stop":1680611778380,"duration":0},"status":"failed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"uncaught exception ScriptExternalLoadError: Loading script failed.\n(error: http://localhost:3005/remoteEntry.js)\nwhile loading \"./components\" from webpack/container/reference/app_05","time":{"start":1680611778380,"stop":1680611778380,"duration":0},"status":"failed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"attachments":[],"parameters":[],"stepsCount":3,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true}],"attachments":[{"uid":"39da71450c38305c","name":"e2e/runAllComprehensiveDemoReact16Tests.cy.ts:2023-04-04T12:36:17.566Z","source":"39da71450c38305c.png","type":"image/png","size":157286},{"uid":"6175ab0ff76b069e","name":"e2e/runAllComprehensiveDemoReact16Tests.cy.ts:2023-04-04T12:36:18.003Z:attempt-1","source":"6175ab0ff76b069e.png","type":"image/png","size":163033},{"uid":"b07d5233e8334478","name":"video recording","source":"b07d5233e8334478.mp4","type":"video/mp4","size":2967019}],"parameters":[],"stepsCount":4,"attachmentsCount":3,"shouldDisplayMessage":false,"hasContent":true},"afterStages":[],"labels":[{"name":"parentSuite","value":"Comprehemsive Demo React 16"},{"name":"suite","value":"Check is Comprehensive Demo App1 working and have elements"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":true,"retry":true,"extra":{"categories":[],"tags":[]},"source":"f249454378ef7a1e.json","parameterValues":[]}